common.sh

```diff
@@ -1,31 +1,59 @@
 
 CACHE=/var/cache/preupgrade
+# Directory with logs gathered by preupgrade-assistant
 PREUPGRADE_CACHE=/var/cache/preupgrade/common
+# Preupgrade-assistant configuration file
 PREUPGRADE_CONFIG=/etc/preupgrade-assistant.conf
+# Full path log file with to all installed packages
 VALUE_RPM_QA=$PREUPGRADE_CACHE/rpm_qa.log
+# Full path log file with to all changed files
 VALUE_ALL_CHANGED=$PREUPGRADE_CACHE/rpm_Va.log
+# Full path to log file with all /etc changed configuration files
 VALUE_CONFIGCHANGED=$PREUPGRADE_CACHE/rpm_etc_Va.log
+# Full path to log file with all users gathered by getent
 VALUE_PASSWD=$PREUPGRADE_CACHE/passwd.log
+# Full path to log file with all services enabled/disabled on system.
 VALUE_CHKCONFIG=$PREUPGRADE_CACHE/chkconfig.log
+# Full path to log file with all groups gathered by getent
 VALUE_GROUP=$PREUPGRADE_CACHE/group.log
+# Full path to log file with all installed files
 VALUE_RPMTRACKEDFILES=$PREUPGRADE_CACHE/rpmtrackedfiles.log
+# Full path to log file with all Red Hat signed packages
 VALUE_RPM_RHSIGNED=$PREUPGRADE_CACHE/rpm_rhsigned.log
+# Full path to log file with all local files
 VALUE_ALLMYFILES=$PREUPGRADE_CACHE/allmyfiles.log
+# Full path to log file with all executable files
 VALUE_EXECUTABLES=$PREUPGRADE_CACHE/executable.log
+# Variable which referes to temporary directory directory provided by module
 VALUE_TMP_PREUPGRADE=$XCCDF_VALUE_TMP_PREUPGRADE
+# postupgrade directory used by in-place upgrades.
+# Scripts mentioned there are executed automatically by redhat-upgrade-tool
 POSTUPGRADE_DIR=$VALUE_TMP_PREUPGRADE/postupgrade.d
 CURRENT_DIRECTORY=$XCCDF_VALUE_CURRENT_DIRECTORY
+# MIGRATE means if preupg binary was used with `--mode migrate` parameter
+# UPGRADE means if preupg binary was used with `--mode upgrade` parameter
+# These modes are used if `--mode` is not used
 MIGRATE=$XCCDF_VALUE_MIGRATE
 UPGRADE=$XCCDF_VALUE_UPGRADE
+# Variable which referes to solution file provided by module
 SOLUTION_FILE=$CURRENT_DIRECTORY/$XCCDF_VALUE_SOLUTION_FILE
+# Directory which is used for kickstart generation
 KICKSTART_DIR=$VALUE_TMP_PREUPGRADE/kickstart
+# README file which contains description about all files in kickstart directory
 KICKSTART_README=$KICKSTART_DIR/README
+# Directory with scripts which can be executed after installation by administrator
 KICKSTART_SCRIPTS=$KICKSTART_DIR/scripts
+# The same as above
 KICKSTART_POSTUPGRADE=$KICKSTART_SCRIPTS
+# Variable which refers to static data used by preupgrade-assistant and modules
 COMMON_DIR=$XCCDF_VALUE_REPORT_DIR/common
 DIST_NATIVE=$XCCDF_VALUE_DIST_NATIVE
+# Variable which indicates DEVEL mode.
 DEVEL_MODE=$XCCDF_VALUE_DEVEL_MODE
+# Variable which contains file with packages add to the kickstart anyway
```

Comment:
please don't write state the obvious: `Variable which contains` when documenting variable and of course it will contain something ;)